Makefile

```diff
@@ -132,8 +132,10 @@ ifeq ($(VERBOSE), true)
 	GO_TEST_FORMAT=standard-verbose
 endif
 
-GOTEST_FLAGS = --format=$(GO_TEST_FORMAT) $(GOTEST_PUBLISH_FLAGS) -- -count=1 -cover -coverprofile=$(REPORTS)/$(TEST_SCENARIO)_coverage.out
-GINKGO_FLAGS = -ginkgo.focus="$(FOCUS)" -ginkgo.v -ginkgo.skip="$(SKIP)" -ginkgo.v -ginkgo.reportFile=./junit_$(TEST_SCENARIO)_test.xml
+COVER_PROFILE:= $(or ${COVER_PROFILE},$($(REPORTS)/$(TEST_SCENARIO)_coverage.out))
+GINKGO_REPORTFILE := $(or $(GINKGO_REPORTFILE), ./junit_$(TEST_SCENARIO)_test.xml)
+GOTEST_FLAGS = --format=$(GO_TEST_FORMAT) $(GOTEST_PUBLISH_FLAGS) -- -count=1 -cover -coverprofile=$(COVER_PROFILE)
```

Comment:
in the jenkins build, seems like it comes out as ``-coverprofile=`` instead of ``-coverprofile=<path>``

Reply:
Looks like `REPORTS` and `TEST_SCENARIO` are not set in the Jenkins builds

Reply:
Fixed `REPORTS`. It was not getting resolved due to extra `$()` around it. `TEST_SCENARIO` still needs to be fixed.